src/orders/integration/models.py

```diff
@@ -21,6 +21,7 @@ class ShopifyCustomer(BaseModel):
 
 
 class ShopifyLineItem(BaseModel):
+    sku: StrictStr | None = None
```

Comment:
this should be mandatory, otherwise we could have a mismatch with Shopify, what do u think?

Reply:
This is an issue cause SKU is not mandatory on shopify config and currently there are several products without SKU. If we set it mandatory here , client wouldn't be aware and order would be lost for delivery. 
Im planning to catch it on orders.errors side  (https://infinity-cloud.atlassian.net/browse/HPDO-30)  so that client is aware through  UI.

Reply:
This is an issue cause SKU is not mandatory on shopify config and currently there are several products without SKU. If we set it mandatory here , client wouldn't be aware and order would be lost for delivery.
Im planning to catch it on orders.errors side (https://infinity-cloud.atlassian.net/browse/HPDO-30) so that client is aware through UI.